pkgs/top-level/all-packages.nix

```diff
@@ -5954,7 +5954,8 @@ in
   inherit (callPackages ../development/libraries/libwebsockets { })
     libwebsockets_3_1
```

Comment:
can we drop 3.1 or 3.2?

Reply:
3.1 is still used by mosquitto. Also, for the 2.x reelase (#110863).